src/sentry/api/endpoints/release_thresholds/health_checks/is_new_issues_count_healthy.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from datetime import datetime
+from logging import getLogger
+
+from sentry import search
+from sentry.api.event_search import SearchFilter, SearchKey, SearchValue
+from sentry.models.project import Project
+from sentry.models.release import Release
+from sentry.models.release_threshold import ReleaseThreshold
+from sentry.models.release_threshold.constants import TriggerType
+from sentry.search.events.constants import RELEASE_ALIAS
+
+default_logger = getLogger("sentry.releases.servicer")
+
+
+def get_new_issue_count_is_healthy(
+    project: Project,
+    release: Release,
+    release_threshold: ReleaseThreshold,
+    start: datetime,
+    end: datetime,
+) -> bool:
+    query_kwargs = {
+        "projects": [project],
+        "date_from": start,
+        "date_to": end,
+        "count_hits": True,
+        "limit": 1,  # we don't need the returned objects, just the total count
```

Comment:
limit doesn't determine whether we return an object at all right?
This just limits the number of results?

Reply:
Nope, just the amount of query rows returned